specification/billing/resource-manager/Microsoft.Billing/preview/2024-08-01-preview/migration.json

```diff
@@ -0,0 +1,148 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "BillingManagementClient",
+    "description": "Documentation for Microsoft.Billing.",
+    "contact": {
+      "name": "Microsoft.Billing",
+      "url": "https://learn.microsoft.com/en-us/rest/api/billing/",
+      "email": "pacebpexphot@microsoft.com"
+    },
+    "version": "2024-08-01-preview"
+  },
+  "paths": {
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/migrations/default": {
+      "get": {
+        "tags": [
+          "Migration"
+        ],
+        "description": "Gets migration mapping details.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/billing/"
+        },
+        "operationId": "Migration_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "./types.json#/parameters/billingAccountName"
+          },
+          {
+            "$ref": "./types.json#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Migration mapping details.",
+            "schema": {
+              "$ref": "#/definitions/MigrationMappingResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get migration mapping details": {
+            "$ref": "./examples/GetMigrationMapping.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Migration"
+        ],
+        "description": "Migrates from source to destination billing scope",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/billing/"
+        },
+        "operationId": "Migration_Put",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "./types.json#/parameters/billingAccountName"
+          },
+          {
+            "$ref": ".,/types.json#/parameters/apiVersionParameter"
+          },
```

Comment:
Have you considered leveraging common types?

Reply:
Yes. "x-ms-parameter-location": "method" is missing in common type. Hence we have our version of apiVersionParameter